02/Form.js

```diff
@@ -2,10 +2,11 @@ import React from 'react';
 
 class Form extends React.Component {
     render() {
+        const { onClick } = this.props;
         return (
-            <section>
-                <input /><input type="submit" />
-            </section>
+            <form onSubmit={onClick}>
```

Comment:
Można to było zrobić też bez dopisywania `<form>` - tylko informacyjnie, opcja z `onSubmit` jest lepsze